### PR TITLE
Execution workflow: DaPars - report Format 04 BED file in workflow (per-PAS fractional relative usage)

### LINKED ISSUE #384: Execution workflow: DaPars - report Format 04 BED file in workflow (per-PAS fractional relative usage)
Parent issue - https://github.com/iRNA-COSI/APAeval/issues/382

Per updated execution workflow [output specifications](https://github.com/iRNA-COSI/APAeval/blob/main/execution_workflows/execution_output_specification.md), We need DaPars to report the per-PAS fractional relative usage in a format 04 BED file.

For each input transcript model, DaPars reports a 'PDUI' (% distal usage index), representing the fraction/percent the distal site is expressed relative to the total expression of proximal and distal site of that locus. Since DaPars2 uses a 2-site model, we can also report the usage of the proximal site by 1 - PDUI.

### PR DESCRIPTION
**Pull request by faricazjj**
Fixes #384

Workflow successfully ran to get relative usage quantification challenge output per sample.

Workflow:

Screen Shot 2022-07-22 at 6 01 11 PM

Example output, the 5th column is the relative usage:
Screen Shot 2022-07-22 at 6 22 17 PM

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated corresponding READMEs (if applicable)
- [ ] My code follows the templates/style guidelines of the repository
- [ ] In- and output formats comply with APAeval specifications
- [ ] No parameters or file names are hardcoded
- [ ] Results, logs or other output is not commited to the repository
- [ ] I have tested my code

### CODE REVIEW

File: execution_workflows/DaPars/README.md [hunk 1]
```diff
@@ -65,18 +65,24 @@ Parameters relevant to the workflow itself are:
...
 When using the default output_dir parameter value in conf/modules.config, DaPars store results under
-DaPars/results/dapars/challenge_outputs folder. For identification 
+DaPars/results/dapars/challenge_outputs folder. For identification and relative usage quantification
 outputs, the files have sample names as prefixes to differentiate the different runs. The differential output file 
 will stay as the name specified in modules.config file.
 
 ## Notes
-- It is not possible to obtain quantification challenge output data since the TPM value in the output file
   is provided per transcript instead of per site. Hence, this tool is not compatible for quantification
   challenge.
```
Comment (SamBryce-Smith) [inline execution_workflows/DaPars/README.md]: If could remove these two lines too would be great :)

**Review by SamBryce-Smith — APPROVED** [review]
LGTM thanks @faricazjj! I noticed a small bug when also running the identification workflow. I added a quick fix and also removed an orphan sentence from the notes section in the README. Good to go on my end! :rocket: 

Identification bug:
dapars-identification-tuple-error-2022-07-25_11-27-23

Successful test run with my quick fix:
dapars-with-list-fix-test-run-Screenshot from 2022-07-25 11-28-42


Rel quant BED looks great. I think at some point we should decide what 'precision' to report these fractions to. For DaPars2 I chose to round proximal to 2 d.p. as DaPars2 only reports the PDUI to 2 d.p.. This isn't really that big of deal right now though, we can always standardise in the summary workflow...
dapars-rel-quant-example-2022-07-25 11-32-14

File: execution_workflows/DaPars/README.md (hunk 1 above)
Comment (SamBryce-Smith) [reply execution_workflows/DaPars/README.md]: removed in b06f5e1

**Review by yuukiiwa — APPROVED** [review]
Thank you so much, @faricazjj!! Everything looks good!

**Comment by faricazjj** [comment]
@SamBryce-Smith @yuukiiwa  thanks so much you guys!! :D

**Pull request merged**